cli.js

```diff
@@ -46,12 +46,12 @@ if (cli.input.length === 0) {
 } else {
 	const promise = fkill(cli.input, {...cli.flags, ignoreCase: true});
 
-	if (!cli.flags.force) {
+	if (cli.flags.silent) {
```

Comment:
Why don't you just add the check for `force` here https://github.com/sindresorhus/fkill-cli/blob/master/cli.js#L51?

```js
if (cli.flags.silent || cli.flags.force) {
    return;
}
```